spec/spec_helper.rb

```diff
@@ -6,7 +6,6 @@
 require './spec/support/helpers'
 
 RSpec.configure do |config|
-  config.treat_symbols_as_metadata_keys_with_true_values = true
   config.run_all_when_everything_filtered = true
```

Comment:
config.treat_symbols_as_metadata_keys_with_true_values is now the default and setting it to false would have no effect either, so it can be removed